docs/main/Overview/Introduction.md

```diff
@@ -12,13 +12,27 @@ A few cool highlights about Hero:
 
 ## Installation
 
-```shell script
+To get started using Hero in your project, we have a "playground" that allows you to run examples out of the box. It can be installed using the following commands:
+
+```bash
+npm i --save @ulixee/hero-playground
+```
+
+or
+
+```bash
+yarn add @ulixee/hero-playground
+```
+
+Once you've completed setting up a local (or remote) [`@ulixee/server`](/docs/server) instance, you can migrate your code to use the production-ready client library:
+
```

Comment:
Isn't the "playground" meant to be more than just a helper for running @ulixee/server? I thought we were planning to tie in helper files behind the scenes for running examples... i.e., for stepping you through the Getting Started tutorial, etc.

Perhaps we should replace the paragraph with something like this: 

> You can drop the "-playground" whenever you want and use hero directly (the core functionality is exactly the same). Just make sure your Ulixee development environment is setup and ready to go, such as making sure @ulixee/server is installed and running.